pom.xml

```diff
@@ -35,20 +35,22 @@
 	<description>Open source RuneScape client</description>
 	<url>http://runelite.net</url>
 
-	<properties>
```

Comment:
remove all modifications to this file.

Reply:
Reverted

Reply:
I'd recommend looking at the diff between what you are pring and master - https://github.com/runelite/runelite/pull/14661/files - you can see that the changes haven't been reverted